sources/platform/storage/request_queue.md

```diff
@@ -274,6 +274,193 @@ Example payload:
 
 For further details and a breakdown of each storage API endpoint, refer to the [API documentation](/api/v2#/reference/key-value-stores).
 
+## Features {#features}
+
```

Comment:
This

> Request queues in Apify power complex web crawling and data scraping operations with a set of robust features.
These features boost your projects' flexibility, efficiency, and scalability.

sounds like a marketing bull-💩 and a bit like GPT. Let's keep the docs accurate and technical with something more like:

> Request queue as a storage type was built with scraping in mind to enable developers to write scraping logic in an efficient and scalable way. The Apify tooling, including Crawlee and Apify SDKs, incorporates all these features, enabling users to leverage them effortlessly without extra configuration.
> 
> In the following section, we will discuss each of the main features in depth.